docs/data/charts/composition/BasicComposition.tsx

```diff
@@ -23,8 +23,7 @@ export default function BasicComposition() {
         labelPlacement="end"
       />
 
-      <Paper sx={{ width: '100%', height: 300 }} elevation={3}>
-        {/* @ts-ignore */}
+      <Paper sx={{ width: '95%', height: 300 }} elevation={3}>
```

Comment:
Don't we want to use something like flex in the parent and do `flex: 1` here? 

Otherwise on smaller screens the `FormControlLabel` might occupy more than 5% of the width, can't it not?

Reply:
I moved from 100% to 95% to let the shadow fo the Paper be visible. But it seems the 100% is not necessary at all, and a `margin: 1` does the job :+1:

![image](https://github.com/user-attachments/assets/3ca9fdd4-9229-4de1-ab13-88bd5e7dafd2)